### ISSUE TITLE
Form--functional

### ISSUE THREAD

**Issue by sabrina-code** [stephanie-ice/Project-1#4]
I will be doing research on how to do a functional form if it is okay with you guys while you are busy with planning on the project-1.

**Comment by stephanie-ice** [comment]
cool! I figured out how to "assign" it to you. I'm just learning zen hub and playing with it today.

**Comment by sabrina-code** [comment]
I tried several ways to do the form, Javascript, Javascript with Php, and 3rd party tools and services. Javascript alone will not work due to client-side script limitations; it does not send out emails. For Js with PHP (a language we have not learned), I've managed to get to half-way, but it may not hit the deadline. Therefore, the last resource is to use 3rd party tools.  However, they charge fees except for Google Forms so far as I could find. Through Google Forms, we can receive email notification about a user's input, and send thank-you response. It is secured, and best of yet is free. All these hassles are because submitting a form needs security to avoid spamming and hacking. Here is where PHP validation comes in. But it takes too long the time to do two-way verifications.
